### PR TITLE
Updating plugins_catalog.md from plugins.yml

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

File: docs/PLUGIN_CATALOG.md [hunk 1]
```diff
@@ -52,15 +52,17 @@ This file is automatically generated. If you would like to add to the plugin lis
...
 | [RabbitMQ](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq) | [Intel](https://github.com/intelsdi-x) | Collects RabbitMQ metrics from management API |  | <ul><li>[release](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq/releases/latest)</li><li>[linux/x86_64](https://s3-us-west-2.amazonaws.com/snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-rabbitmq/latest/linux/x86_64/snap-plugin-collector-rabbitmq)</li></ul> |
+| [SNMP](https://github.com/intelsdi-x/snap-plugin-collector-snmp) | [Intel](https://github.com/intelsdi-x) | Collects metrics using SNMP. | <ul><li>[![Build Status](https://travis-ci.org/intelsdi-x/snap-plugin-collector-snmp.svg?branch=master)](https://travis-ci.org/intelsdi-x/snap-plugin-collector-snmp)</li></ul> | <ul><li>[release](https://github.com/intelsdi-x/snap-plugin-collector-snmp/releases/latest)</li><li>[linux/x86_64](https://s3-us-west-2.amazonaws.com/snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-snmp/latest/linux/x86_64/snap-plugin-collector-snmp)</li></ul> |
```
Comment (kindermoumoute) [inline docs/PLUGIN_CATALOG.md]: Why SNMP doesn't follow alphabetical order anymore?

**Comment by kjlyon** [comment]
Why does our Mesos plugin not have a download link for latest? Also, one of the Docker CI tests is doing something strange. See picture. The link leads to a 404 Page Not Found.
screen shot 2016-12-09 at 6 08 02 pm

File: docs/PLUGIN_CATALOG.md (hunk 1 above)
Comment (nanliu) [inline docs/PLUGIN_CATALOG.md]: Appears I need to downcase before the sort:
```
=> ['a', 'A', 'B', 'b', 'c'].sort
=> ["A", "B", "a", "b", "c"]
```

**Comment by nanliu** [comment]
We need to cut a new release for mesos before the download link is valid.  The docker build is no longer valid because I removed the large test (since Docker-in-Docker doesn't work in Jenkins). I need to also remove it from the docker plugin repo, so I'll take care of that before regenerating this PR.

**Pull request closed without merging**